dynatrace-aws-log-forwarder-template.yaml

```diff
@@ -518,6 +563,128 @@ Resources:
         - __Region__: !Ref 'AWS::Region'
           __LambdaName__: !Ref Lambda
 
+  EC2ActiveGate:
+    Condition: DeployVPCwithAG
+    Type: AWS::EC2::Instance
+    DependsOn: VPCPublicRouteInternet
+    Properties:
+      ImageId: !Ref LatestAmazonLinuxAmiId
+      InstanceType: "t3.small"
```

Comment:
Is it enough for production use? Or was it only used for your tests?

Reply:
I was asking myself the same question. 
I was also wondering if we allow the customer to explicitly choose different/bigger machine size if they don't like the default. But I got stuck on checking what Azure has.

Reply:
![image](https://user-images.githubusercontent.com/16317514/122553366-dd544c00-d037-11eb-8594-c23a171d6899.png)


This one has 2vCPU and 2GB ram.
You can't install AG with less than 2GB ram. You can start AG in container, though.

Reply:
I did not test it with large workloads but would assume it is a production size, in AG docs 2GB is minimum memory. (4GB is recommended but our AG only pushes logs, all other modules are disabled, so not very overworked and should be fine)